Gemfile

```diff
@@ -10,10 +10,7 @@ source "https://rubygems.org"
 # This is the default theme for new Jekyll sites. You may change this to anything you like.
 gem "minima", "~> 2.0"
```

Comment:
Should remove this too?

Reply:
Yes, removing the line has no impact. Removed in a new commit.